### PR TITLE
Add labels and hover feedback for content download buttons

### PR DESCRIPTION
**Pull request by dylanmccall**
Closes https://github.com/endlessm/kolibri-explore-plugin/issues/684

### CODE REVIEW

File: packages/ek-components/src/components/EkPlayButton.vue [hunk 1]
```diff
@@ -114,7 +114,7 @@ export default {
 }
 
 button:disabled {
-  opacity: 33%;
+  opacity: 0.5;
```
Comment (manuq) [inline packages/ek-components/src/components/EkPlayButton.vue:117]: :+1:

File: packages/ek-components/src/components/EkCardButtons.vue [hunk 2]
```diff
@@ -127,6 +139,14 @@ export default {
 <style lang="scss" scoped>
 @import '../styles.scss';
 
+.card-buttons {
+  &::after {
+    content: '';
+    display: block;
+    clear: both;
+  }
+}
```
Comment (manuq) [inline packages/ek-components/src/components/EkCardButtons.vue:148]: Nit: this change has nothing to do with the commit message.

**Review by manuq — APPROVED** [review]
All looks good to me! Only a nit observed.

File: packages/ek-components/src/components/EkCardButtons.vue (hunk 2 above)
Comment (dylanmccall) [reply packages/ek-components/src/components/EkCardButtons.vue:148]: Oops, that should be in its own _issue_, even: Joana and I noticed the margins were inconsistent when we were looking at this. Good thing to keep track of properly :) I'll move that change elsewhere for orderliness and then merge this. Thank you for the review!

Comment (dylanmccall) [reply packages/ek-components/src/components/EkCardButtons.vue]: I split this into https://github.com/endlessm/kolibri-explore-plugin/issues/690.

**Pull request merged**